src/components/UserProfile/VolunteeringTimeTab/VolunteeringTimeTab.jsx

```diff
@@ -19,6 +21,9 @@ const StartDate = (props) => {
       onChange={(e) => {
         props.setChanged(true);
         props.setUserProfile({ ...props.userProfile, createdDate: e.target.value });
+        <Alert color="warning">
```

Comment:
I think we can do a bit refactoring here. I removed all other `<Alert />` elements just kept the last one, it also worked.